netbox/ipam/filtersets.py

```diff
@@ -599,8 +600,24 @@ def search_by_parent(self, queryset, name, value):
                 return queryset.none()
         return queryset.filter(q)
 
+    def parse_inet_addresses(self, value):
+        try:
+            parsed = []
+            for addr in value:
+                if netaddr.valid_ipv4(addr) or netaddr.valid_ipv6(addr):
+                    parsed.append(addr)
+                    continue
+                network = netaddr.IPNetwork(addr)
+                parsed.append(str(network))
+            return parsed
+        except (AddrFormatError, ValueError):
+            raise serializers.ValidationError({
```

Comment:
We probably need to avoid raising any validation errors inside the filter. This works for API requests, but the filter is used for UI requests as well, which don't catch the exception. In the event an invalid address is found, we can just exclude it from the list of values. (If the invalid address is the only value provided, this should result in returning an empty results list.)

Reply:
Got it - Will change to silently drop invalid addresses and potentially return an empty list if all the addresses provided are invalid.